lib/jekyll/site.rb

```diff
@@ -6,9 +6,9 @@ class Site
     attr_reader   :source, :dest, :config
     attr_accessor :layouts, :pages, :static_files, :drafts,
                   :exclude, :include, :lsi, :highlighter, :permalink_style,
-                  :time, :future, :unpublished, :safe, :plugins, :limit_posts,
+                  :time, :future, :unpublished, :safe, :plugins_dir, :limit_posts,
                   :show_drafts, :keep_files, :baseurl, :data, :file_read_opts,
-                  :gems, :plugin_manager, :theme
+                  :plugins, :plugin_manager, :theme
```

Comment:
I've the same concern as @parkr, we should probably alias the old values over until 4.0?